### PR TITLE
feat: added option to set null cell display and type to be date or cu…

### PR DESCRIPTION
**Pull request by jmy-kellogg**
…rrency

BREAKING CHANGE: isDate should become type:'date'

### CODE REVIEW

File: src/components/DefaultTable.vue [hunk 1]
```diff
@@ -138,14 +144,22 @@ export default {
...
+        const value = parseInt(cellValue)
+        return currencyFormatter.format(value, {
+          code: row.currency || null
+        })
       } else {
         return cellValue
       }
-    }
+    },
+    getCurrencyFormat (amount) {
```
Comment (Endebert) [inline src/components/DefaultTable.vue]: Is there a specific reason why you kept this empty function? It's unused as well

File: src/components/VueTablePage.vue [hunk 2]
```diff
@@ -162,12 +162,12 @@ export default {
   },
   data () {
     return {
-      show: this.hideInfoBtn ? true : this.showMessage,
+      show: this.message.length ? this.showMessage : false,
```
Comment (Endebert) [inline src/components/VueTablePage.vue]: Clever, although I would never use a number as a condition like this and would still do the explicit `===  0` check. Don't know if your implementation is considered bad practice or not (or if it has performance benefits).

File: src/components/DefaultTable.vue (hunk 1 above)
Comment (jmy-kellogg) [reply src/components/DefaultTable.vue]: nope. I think it was me going to do something cleaver then not, and not cleaning up my code. Gone now

File: src/components/VueTablePage.vue (hunk 2 above)
Comment (jmy-kellogg) [reply src/components/VueTablePage.vue]: It's not uncommon to use 0, undefined and so on as falsey. I think the laxness is due to the javascript language. I think now it just a preference of coding style.

**Pull request merged**